src/bin/travis/resources/tes.conf

```diff
@@ -7,9 +7,13 @@ Storage:
         - /cromwell-executions
         - /tmp/
 DBPath: /tmp/tes_task.db
-Schedulers:
-  Local:
-    NumWorkers: 4
+Scheduler: local
 Worker:
   LogLevel: info
-  Timeout: 1
+  Timeout: -1
+  # Funnel (TES implementation) respects resource reqs
+  # Defaults 1 cpu 2 GB ram would make centaur take too long to run
+  Resources:
+    cpus: 100
+    ram: 200
+    disk: 1000
```

Comment:
OOI, where does this get provisioned from... I ask because are we going to be creating this somewhere every time we run the TES tests in Travis?